crates/bevy_ptr/src/lib.rs

```diff
@@ -116,13 +116,9 @@ impl<'a> Ptr<'a> {
     ///
     /// If possible, it is strongly encouraged to use [`deref`](Self::deref) over this function,
     /// as it retains the lifetime.
-    ///
-    /// # Safety
-    /// All subsequent operations to the returned pointer must be valid inside the
-    /// associated lifetime.
     #[inline]
     #[allow(clippy::wrong_self_convention)]
-    pub unsafe fn as_ptr(self) -> *mut u8 {
+    pub fn as_ptr(self) -> *mut u8 {
```

Comment:
this isnt taking `&self`

Reply:
it probably doesnt matter but idk if we want to be consistent here or not :sweat_smile: